packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -645,7 +664,7 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 				  };
 
 		try {
-			while (opsSent < clientSendCount && !this.disposed) {
+			while (dataModel.counter.value < clientSendCount && !this.disposed) {
```

Comment:
Shouldn't this be `dataModel.counter.value < config.testConfig.totalSendCount`? dataModel.counter is the collective count of ops sent by all client. So, it needs to be compared against the totalSendCount.